build/scripts/Testing.fs

```diff
@@ -54,7 +53,10 @@ module Tests =
             | (true) -> [ "--logger"; "trx"; "--collect"; "\"Code Coverage\""; "-v"; "m"] |> List.append command
             | _  -> command
             
-        Tooling.DotNet.ExecInWithTimeout "src/Tests/Tests" commandWithCodeCoverage (TimeSpan.FromMinutes 30.) 
+        // using std redirection since that foces vstest not to redirect Console.Write from Elastic.Xunit somehow
```

Comment:
Should this comment be updated with how vstest and xunit communicate?

Reply:
reverting this since `Proc` is not happy on linux